test/NuGet.Core.Tests/NuGet.DependencyResolver.Core.Tests/RemoteDependencyWalkerTests.cs

```diff
@@ -1027,6 +1027,49 @@ public void IsDependencyValidForGraphTest(LibraryDependencyReferenceType referen
             }
         }
 
+        [Fact]
+        public async Task WalkAsyncCentralDependencyDoesNotHaveDuplicates()
```

Comment:
naming: Not clear to me what the set-up is. 
Try changing it like: 
`MethodName_StateUnderTest_ExpectedBehavior`

Reply:
Changed the names.